README.rst

```diff
@@ -217,8 +218,30 @@ Viewing the logs
 Running the Tests
 """""""""""""""""
 
-To run the tests via Tox, use this command.
+To run the tests via Tox, use this command. If you are using podman-compose, swap the word docker with podman (see ``Developing with Podman and Podman-Compose`` below).
```

Comment:
For the three longer lines that are added in this README, I would break them up, so the lines are less than 100 characters.